packages/libretro/sameboy/package.mk

```diff
@@ -25,6 +25,7 @@ PKG_ARCH="any"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/libretro/sameboy"
 PKG_URL="https://github.com/libretro/sameboy/archive/$PKG_VERSION.tar.gz"
+PKG_SOURCE_DIR="SameBoy-$PKG_VERSION*"
```

Comment:
* ?

Reply:
If you don't specify the name of the directory that gets unpacked from the tarball, it guesses (in this case incorrectly) from the package name.